src/main/java/org/folio/circulation/domain/ItemStatus.java

```diff
@@ -11,7 +11,9 @@
   CHECKED_OUT("Checked out"),
   IN_TRANSIT("In transit"),
   MISSING("Missing"),
-  PAGED("Paged");
+  PAGED("Paged"),
+  ON_ORDER("On order"),
+  IN_PROCESS("In Process");
```

Comment:
The typical capitalisation in FOLIO is only the first word of a phrase. And the [examples in mod-orders](https://github.com/folio-org/mod-orders/blob/223580178d46c45d13479db68fc350f521b7da6a/src/test/java/org/folio/rest/impl/OrdersImplTest.java#L3665) seem to reflect this.

Was `In Process` specified this way in the story?